OpenRA.Game/Network/Session.cs

```diff
@@ -69,6 +71,9 @@ public static Session Deserialize(string data)
 							var s = Slot.Deserialize(node.Value);
 							session.Slots.Add(s.PlayerReference, s);
 							break;
+						case nameof(DisabledSpawns):
```

Comment:
We'll most likely want to use a string literal, like the other cases.
See my comment regarding the `LangVersion` change in `OpenRA.Game.csproj`.